Changes to Gmail Policies 9.1, 9.2 and 13.1 (#310)
cisagov/ScubaGoggles: baselines/Gmail Minimum Viable Secure Configuration Baseline v0.2.md

To configure the settings for POP and IMAP access:

#### GWS.GMAIL.9.1v0.2 Instructions
1.  Sign in to the [Google Admin Console](https://admin.google.com).
2.  Select **Apps -\> Google Workspace -\> Gmail**.
3.  Select **End User Access -\> POP and IMAP access**.

> 4.  Select **Save**.
> 5.  Uncheck the **Enable IMAP access for all users** checkbox.
> 6.  Uncheck the **Enable POP access for all users** checkbox.

Reviewer: One more thing, "Save" should be the last step.
```suggestion
4.  Uncheck the **Enable IMAP access for all users** checkbox.
5.  Uncheck the **Enable POP access for all users** checkbox.
6.  Select **Save**.
```

To configure the settings for POP and IMAP access:

#### GWS.GMAIL.9.1v0.2 Instructions
1.  Sign in to the [Google Admin Console](https://admin.google.com).
2.  Select **Apps -\> Google Workspace -\> Gmail**.
3.  Select **End User Access -\> POP and IMAP access**.

> 4.  Follow the implementation for each policy.

Reviewer: "Follow the implementation for each policy" needs to be cut as there's only one policy now, and the steps following this one renumbered.

### Policies

#### GWS.GMAIL.13.1v0.2

> Unintended external reply warnings SHALL be enabled

Reviewer: Policy statements should always end with a period.
```suggestion
Unintended external reply warnings SHALL be enabled.
```

> - _Rationale:_ Unintended external reply warnings can help reduce the risk of exposing sensitive information in replies to external messages. Enabling these warnings reminds users to treat external messages with caution, reducing this risk and enhancing the safety and integrity of user data and systems. This helps avoid unintentional replies and remind users to treat external messages with caution.

Reviewer: The rationale already says that this reminds "reminds users to treat external messages with caution," no need to state it twice.

```suggestion
- _Rationale:_ Unintended external reply warnings can help reduce the risk of exposing sensitive information in replies to external messages. Enabling these warnings reminds users to treat external messages with caution, reducing this risk and enhancing the safety and integrity of user data and systems. Additionally, this may help avoid unintentional replies.
```
